angular-spa-ui/js/search-app/search-ctrl.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+module.exports = function(app) {
+    
+    app.controller('mainCtrl', function( $scope, promises, $q, getTemplate, appConfig) {
+
+        var config = appConfig.config;
+        var path = config.path;
+        var methods = config.methods;
+        var userData = config.userData;
+        
+        path.getUserPath = path.getUserPath + GLOBAL_USER_ID;
+        
+        function initUserData(data) {
+            userData.responsePromises.push(data);
+            userData.userId = data.data._id;
+            path.getRolePath = path.getRolePath + userData.userId;
+            path.getTokenPath = path.getTokenPath + userData.userId;
+        };
+
+        function initRTData(data) {
+            for (var x in data){
```

Comment:
Are you sure that structure for (var item in items) is applicable for ECMA Script 5?